atd-vze/src/containers/DefaultLayout/DefaultHeader.js

```diff
@@ -44,11 +45,17 @@ const DefaultHeader = props => {
             Dashboard
           </NavLink>
         </NavItem>
-        <NavItem className="px-3">
-          <NavLink to="/users" className="nav-link">
-            Users
-          </NavLink>
-        </NavItem>
+        <Can
+          roles={getRoles()}
+          perform="users:visit"
+          yes={() => (
+            <NavItem className="px-3">
+              <NavLink to="/users" className="nav-link">
+                Users
+              </NavLink>
+            </NavItem>
+          )}
+        />
```

Comment:
Awesome code, it's exciting to think how this is going to be scaled up for editing. 🙌